extensions/rt_ignore_dh/rt_ignore_dh.c

```diff
@@ -165,8 +165,13 @@ static int stow_destination_host(struct msg **msg) {
 			CHECK_FCT(fd_msg_avp_add(pi_avp, MSG_BRW_LAST_CHILD, ph_avp));
 			CHECK_FCT(fd_msg_avp_add(pi_avp, MSG_BRW_LAST_CHILD, ps_avp));
 
-			/* remove Destination-Host from message */
-			fd_msg_free((msg_or_avp*)avp);
+			/* set the Destination-Host to the local Diameter Identity */
+			union avp_value dh;
+			memset(&dh, 0, sizeof(dh));
+			dh.os.data = memdup(fd_g_config->cnf_diamid, fd_g_config->cnf_diamid_len);
+			dh.os.len = fd_g_config->cnf_diamid_len;
+			CHECK_FCT(fd_msg_avp_setvalue(avp, &dh));
```

Comment:
Doesn't this mean that the response will be sent from the local Diameter identity?  Surely that's wrong?